runner/full_1min.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-func gatherQueryStatsForServer(ctx context.Context, server *state.Server, globalCollectionOpts state.CollectionOpts, logger *util.Logger) (state.PersistedState, error) {
+func gather1minStatsForServer(ctx context.Context, server *state.Server, globalCollectionOpts state.CollectionOpts, logger *util.Logger) (state.PersistedState, error) {
```

Comment:
The `1min` in the function name jumped out a bit at me at first glance, but not sure there is a better name here - using something like "high frequency" in the name is probably more confusing than helpful.